src/website/views/components/toolbar/_properties.pug

```diff
@@ -149,3 +202,12 @@ section.chi-table.chi-table__options.-bordered.-my--3
           td
             div
               code DataTableColumns[]
+        tr
+          td
+            div
+              code chiToolbarViewsChange
+          td
+            div To change the view of your table
```

Comment:
Looks like Toolbar component event descriptions can be improved.
Please, check out Data Table events and use the same format.
The idea is to describe **when** the event is being emitted, like this:


chiDataSorting | The user has sorted the table by a specific column
-- | --